### DISCUSSION TITLE
How to use local storage in quasar electron app ?

### DISCUSSION THREAD

**Discussion by amirsafaricg** [quasarframework/quasar#16733] [CLI - Electron mode]
Hey everyone 
I wanted to store my apps data in local storage but Im not sure how to achieve this in quasar electron app. Can someone guide me through how to use local storage for my data ? 
Should I use reading and writing local files ? Should I use the local storage plugin ? 
Thanks in advance for the help.

**Comment by mecjos** [comment]
You can use web storage plugin

https://quasar.dev/quasar-plugins/web-storage#localstorage-api

It's same with spa usage.
